examples/mail/searchItems.php

```diff
@@ -0,0 +1,82 @@
+<?php
+
+require_once "vendor/autoload.php";
+
+use garethp\ews\API\Enumeration\DistinguishedFolderIdNameType;
+use garethp\ews\API\ExchangeWebServices;
+use garethp\ews\API\Type;
+use garethp\ews\API\Type\DistinguishedFolderIdType;
+
+$ews = ExchangeWebServices::withUsernameAndPassword('server', 'username', 'password');
+
+$request = array(
```

Comment:
If you could add a quick explanation of what messages are being searched for, I think this example would flow more easily

Reply:
Done.